pkg/segment/structs/evaluationstructs.go

```diff
@@ -272,6 +306,13 @@ func (self *BoolExpr) Evaluate(fieldToValue map[string]utils.CValueEnclosure) (b
 	}
 }
 
+func likePatternToRegex(pattern string) string {
+	pattern = regexp.QuoteMeta(pattern)
+	pattern = strings.Replace(pattern, "%", ".*", -1)
+	pattern = strings.Replace(pattern, "_", ".", -1)
+	return "^" + pattern + "$"
+}
```

Comment:
Are you calling this function anywhere?

Reply:
I have incorporated the functionality directly in the case so no need of this function anymore.

Reply:
Ok, just make sure to delete this function then